### PR TITLE
reclaim policy changed to retain since recycle is not supported for g…

### LINKED ISSUE #2736: Not supported Recycle policy in Using GlusterFS
Use Retain as Recycle policy is currently not supported with glusterfs.

https://docs.openshift.org/latest/install_config/persistent_storage/persistent_storage_glusterfs.html#gfs-creating-persistent-volume

### PR DESCRIPTION
**Pull request by jeffvance**
... and some minor additions/corrections. Also fixes #2736 
@adellape @copejon  PTAL

### CODE REVIEW

**Comment by erinboyd** [comment]
LGTM..thanks for catching this and making updates

**Comment by jeffvance** [comment]
@adellape when you get a chance PTAL

**Comment by erinboyd** [comment]
LGTM

File: install_config/storage_examples/binding_pv_by_label.adoc [hunk 1]
```diff
@@ -15,19 +15,19 @@ toc::[]
 == Overview
 This topic provides an end-to-end example for binding
 xref:../../architecture/additional_concepts/storage.adoc#persistent-volume-claims[persistent
-volume claims] (PVCs) to
+volume claims] (Persistent Volume Claims, PVCs) to
 xref:../../architecture/additional_concepts/storage.adoc#persistent-volumes[persistent
-volumes] (PVs) by defining labels in the PV and matching selectors in the PVC.
+volumes] (Persistent Volumes, PVs), by defining labels in the PV and matching selectors in the PVC.
```
Comment (adellape) [inline install_config/storage_examples/binding_pv_by_label.adoc]: Why spell "Persistent Volume Claims" and "Persistent Volumes" out twice each in these two spots?

**Comment by adellape** [comment]
@jeffvance LGTM, thanks! One question inline, but not enough to block merge. Docs will team do minor style edits in follow-up.

File: install_config/storage_examples/binding_pv_by_label.adoc (hunk 1 above)
Comment (jeffvance) [inline install_config/storage_examples/binding_pv_by_label.adoc]: The reason I did that was to equate the abbreviation, PV, to the full name once at the beginning. I'm happy to delete this if you think it's too wordy.

Comment (adellape) [inline install_config/storage_examples/binding_pv_by_label.adoc]: Aren't the full names being spelled out once at the beginning already? In the link text in the square brackets, e.g.:

```
xref:../../architecture/additional_concepts/storage.adoc#persistent-volume-claims[persistent volume claims] (PVCs)
```

Or am I misunderstanding?

Comment (jeffvance) [inline install_config/storage_examples/binding_pv_by_label.adoc]: You're right. I missed that full name as the link.  I see this pr is merged. I can create a new pr for this one change or we can try to catch it the next time the doc is changed. Preference?

Comment (adellape) [inline install_config/storage_examples/binding_pv_by_label.adoc]: @jeffvance I already labeled this PR for follow-up so we'll get it then.

**Comment by ahardin-rh** [comment]
[rev_history]
|xref:../install_config/storage_examples/binding_pv_by_label.adoc#binding-pv-by-label[Persistent Storage Examples -> Binding Persistent Volumes by Labels]
|Updated the `*persistentVolumeReclaimPolicy*` setting to *retain* in the xref:../install_config/storage_examples/binding_pv_by_label.adoc#binding-pv-by-label-pvc-with-selectors[glusterfs-pv.yaml example], since *recycle* is not supported in this case.
%
